services/sourceforge/sourceforge-tickets.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  count: Joi.number().required(),
+}).required()
+
+module.exports = class SourceforgeTickets extends BaseJsonService {
+  static get category() {
+    return 'other'
+  }
+
+  static get route() {
+    return {
+      base: 'sourceforge/tickets',
+      pattern: ':type(bugs|feature-requests)/:project',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Sourceforge Open Tickets',
+        namedParams: {
+          type: 'bugs',
+          project: 'sevenzip',
+        },
+        staticPreview: this.render({ count: 1338 }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'open tickets',
+      color: 'blue',
+    }
+  }
+
+  static render({ count }) {
+    return {
+      message: metric(count),
+    }
+  }
+
+  async fetch({ type, project }) {
+    const url = `https://sourceforge.net/rest/p/${project}/${type}/search?limit=1&q=status:open`
```

Comment:
Think that colon may need to be encoded. Could either do it manually 
```suggestion
    const url = `https://sourceforge.net/rest/p/${project}/${type}/search?limit=1&q=status%3Aopen`
```

Or
```suggestion
    const url = `https://sourceforge.net/rest/p/${project}/${type}/search?limit=1&q=${encodeURIComponent('status:open')}`
```

Reply:
From sourceforge's rules:

```
It must begin with a letter.
It must be between 3 and 15 characters in length.
It can only contain only lowercase letters (a-z), numbers (0-9), and dashes ("-") or underscores ("_")
```

I think we can get away without encoding it.